### PR TITLE
what's the pim marketplace article

### PR DESCRIPTION
**Pull request by meganelepalud-akeneo**
- [x] Article text
- [x] Screenshot

### CODE REVIEW

File: content/md/what-is/what-is-the-pim-marketplace.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+To ease the way to find the right extension to connect your Akeneo PIM to its ecosystem, we developed the PIM `Marketplace`. 
+
+![PIM Marketplace](../img/Pim-marketplace.png)
+
+Thanks to the PIM `Marketplace`, you can see all the connectors available on our Marketplace and compatible with your Akeneo PIM version and edition. When you find a connector that could be interesting, click on `More info` to reach the connector documentation. 
+
+From that page, you'll be able to read all the connector documentation, consult all the available screenshots, reach the connector developer or even download it. 
+
+To find the connectors you may need, go to `Connect`, then click on `Marketplace`, and here you are!
```
Comment (camillebourjade) [inline content/md/what-is/what-is-the-pim-marketplace.md:33]:
```suggestion
To find the connectors you need, go to `Connect`, then click on `Marketplace`, and here you are! 
```

File: content/md/what-is/what-is-the-pim-marketplace.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Thanks to the PIM `Marketplace`, you can see all the connectors available on our Marketplace and compatible with your Akeneo PIM version and edition. When you find a connector that could be interesting, click on `More info` to reach the connector documentation. 
+
+From that page, you'll be able to read all the connector documentation, consult all the available screenshots, reach the connector developer or even download it. 
+
+To find the connectors you may need, go to `Connect`, then click on `Marketplace`, and here you are! 
+
+# What's a connector?
+A connector is a bridge between Akeneo PIM and a third-party component or application. 
+It enables to collect data or spread it to one or several channels.
```
Comment (camillebourjade) [inline content/md/what-is/what-is-the-pim-marketplace.md:37]:
```suggestion
It enables data collection and sharing to one or several channels. 
```

File: content/md/what-is/what-is-the-pim-marketplace.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+From that page, you'll be able to read all the connector documentation, consult all the available screenshots, reach the connector developer or even download it. 
+
+To find the connectors you may need, go to `Connect`, then click on `Marketplace`, and here you are! 
+
+# What's a connector?
+A connector is a bridge between Akeneo PIM and a third-party component or application. 
+It enables to collect data or spread it to one or several channels. 
+
+To comply with the Serenity Edition (full-SaaS mode), Akeneo connectors should rely on our web API and be hosted outside the PIM interface. Akeneo API provides all necessary endpoints to connect the PIM structure and entities with any application.
```
Comment (camillebourjade) [inline content/md/what-is/what-is-the-pim-marketplace.md:39]:
```suggestion
```

**Review by camillebourjade — COMMENTED** [review]
English fix

**Review by alexandrelissowski — APPROVED** [review]

**Pull request merged**